src/cpu/o3/lsq.cc

```diff
@@ -1411,6 +1465,92 @@ LSQ::SplitDataRequest::isCacheBlockHit(Addr blockAddr, Addr blockMask)
 bool
 LSQ::DcachePort::recvTimingResp(PacketPtr pkt)
 {
+    // Throttling enabled
+    if (lsq->recvRespThrottling) {
```

Comment:
For readibility purposes, it would be nice if we could move all this logic on a separate method